test/framework/options.py

```diff
@@ -1898,6 +1898,13 @@ def test_review_pr(self):
         self.mock_stdout(False)
         self.assertTrue(re.search(r"^Comparing zlib-1.2.8\S* with zlib-1.2.8", txt))
 
+    def test_allow_values_that_are_probably_meant_to_be_options(self):
+        """Test detecting typos or ambiguities on the eb command line."""
+        common_args = []
+
+        eb_go = parse_options(args=common_args + ['-r', 'f'])
+        self.assertFalse(eb_go.options.robot == 'f')
```

Comment:
can you add a test that is not store_or_none? just to make sure the vsc-base is up to date? (or is this with `python setup.py test`?)